test/run/float-ops.mo

```diff
@@ -90,3 +90,19 @@ assert (Prim.floatToInt64(-pi) == (-3 : Int64));
 
 assert (Prim.int64ToFloat(42) == 42.0);
 assert (Prim.int64ToFloat(-42) == -42.0);
+
+Prim.debugPrint (debug_show Prim.floatToInt(-pi));
+/*Prim.debugPrint (debug_show Prim.floatToInt(pi));
+Prim.debugPrint (debug_show Prim.intToFloat(42));
+Prim.debugPrint (debug_show Prim.intToFloat(-42));
+Prim.debugPrint (debug_show Prim.intToFloat(-42_000_000_000));*/
+
+assert (Prim.floatToInt(pi) == 3);
+assert (Prim.floatToInt(-pi) == -3);
+
+assert (Prim.intToFloat(42) == 42.0);
+assert (Prim.intToFloat(-42) == -42.0);
+
+
+assert (Prim.intToFloat(42_000_000_000) == 42000000000.0);
```

Comment:
test the compact-to-heap bignum boundary (add Rust debug assertions).

Reply:
Done with f3c7454bc.

Reply:
In Rust we can add a debugging assert to `persist_bigint`, which checks that heap allocated bignums are never tiny.

Reply:
I'll leave this to another PR.